blog/2024-07-31-open-planning-r24.12.mdx

```diff
@@ -0,0 +1,122 @@
+Please feel free to forward this post to anyone interested in contributing, whether in planning or development.
+
+## Open Meetings
+
+- **[Refinement Phase - Release 24.12](#refinement-phase---release-2412)**
+- **[Refinement Day 1 - Release 24.12](#refinement-day-1---release-2412)**
+- **[Draft Feature Freeze - Release 24.12](#draft-feature-freeze---release-2412)**
+- **[Refinement Day 2 - Release 24.12](#refinement-day-2---release-2412)**
+- **[Open Planning Days - Release 24.12](#open-planning-days---release-2412)**
+
+
+<!--truncate-->
+
+### Refinement Phase - Release 24.12
+
+Please be aware that the Refinement Phase is to keep you aware of the overall timeline during the Release Process and does not constitute an actual meeting. It serves informational
+purposes only. During this time, the necessary people/groups should come together and work (refine) the future features. Feature authors take care of organising appointments and
+communicating with the relevant contacts themselves. The Teams Session link can be used. Be aware: The link and the shared content is available to everyone.
+
+- *Monday, 24. June 2024 to Sunday, 28. July 2024 (CEST)*
+- [open meeting](https://eclipse-tractusx.github.io/community/open-meetings/#Refinement%20Phase%20-%20Release%2024.12)
+
+:::info
+The detailed agenda will follow as soon as possible
+:::
+
+### Refinement Day 1 - Release 24.12
+
+This event is organised and moderated by the Catena-X Association. Based on a predefined agenda (depending on the features on the board), individual dependencies are discussed and
+also documented directly on the feature. If the refinement of the features has already been carried out 'in the refinement phase' and the Definition of Entry (DoE) has already reached,
+this meeting can also be used as synchronisation.
+
+- *Monday, 1. July 2024 from 09:05 to 12:00 (CEST)*
+- [open meeting](https://eclipse-tractusx.github.io/community/open-meetings/#Refinement%20Day%201%20-%20Release%2024.12)
+
+:::info
+The detailed agenda will follow as soon as possible
+:::
+
+### Draft Feature Freeze - Release 24.12
+
+This event is organised and moderated by the Catena-X Association. Until this date, feature proposal can be submitted to the planning board (in GitHub). The current status (as an
+interim status in the refinement phase) of the previous feature proposals will also be discussed at this meeting.
+
+- *Friday, 12. July 2024 from 09:05 to 12:00 (CEST)*
+- [open meeting](https://eclipse-tractusx.github.io/community/open-meetings/#Draft%20Feature%20Freeze%20-%20Release%2024.12)
+
+:::info
+The detailed agenda will follow as soon as possible
+:::
+
+### Refinement Day 2 - Release 24.12
+
+This event is organised and moderated by the Catena-X Association. Based on a predefined agenda (depending on the features on the board), individual dependencies are discussed and
```

Comment:
I personally wouldn't mention process details in the announcement, but rather document them in the [process definition in sig-release](https://github.com/eclipse-tractusx/sig-release/blob/main/docs/open-refinement-and-planning.md). 
The announcement should be short and link to further information instead of duplicating

Reply:
Thanks for clarifying! Agreed if the link is provided!

Reply:
Since there are 21 days between, both will take place